spec/spec_helper.rb

```diff
@@ -12,6 +12,14 @@
 require 'factory_girl_rails'
 require File.join(Sufia::Engine.root, 'spec/factories/generic_files')
 require File.join(Sufia::Engine.root, 'spec/support/features')
+
```

Comment:
With Rspec v3, we shouldn't be using the spec_helper for our main test helper. spec_helper.rb is suppose to be a lighter weight test helper for use in a non rails environment (to save on boot time). The rails_helper.rb is suppose to have everything needed for our rails tests.

Also, instead of polluting this file with everything we need to have configured for our tests, these configurations and helpers should live in their own files under support directory. This seems to be a best practice for better code readability. You could even take it a step further and remove this globbing of files defined below and only include the files for certain tests (for example capybara should only be initialised on integration tests for example). Something to think about.